src/main.ts

```diff
@@ -260,6 +260,7 @@ export {
   logicSigFromByte,
   tealSign,
   tealSignFromProgram,
+  LogicSig,
```

Comment:
Also, what is the motivation behind exporting `LogicSig` now? It probably _should_ have been exported all along, but it's now deprecated in favor of `LogicSigAccount`, which is exported.

Reply:
Ah, we missed that it's deprecated. We were using `LogicSig` type for a year and then when we migrated from our generated typescript typing to the new typescript SDK it was missing. We will try to migrate our codebase.